.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -35,11 +35,23 @@
 
 ### Test Coverage
 
-<!-- TODO: Please show how you have added tests to cover your changes,
-  -        or tell us how you tested it. For each systems you tested,
-  -        uncomment the systems in the list below.
+<!-- TODO: Please do one of the following:
+  -    * Create unit tests, and explain here how they cover your changes.
+  -    * List steps you used for manual testing, and explain how they cover
+  -      your changes.
+  -      For each system you manually tested, uncomment it below.
   -->
 
+<!-- I created tests for ... -->
+
+<!-- Steps you used to test:
+  -   1. Go to '...'
+  -   2. Click on '...'
+  -   3. Scroll down to '...'
+  -   4. See error
+  -->
+
+
```

Comment:
I think you should consider removing this long list of browsers, nobody (including team members) actually uses it and the more things a person has to delete from the template, the less likely they are to actually fill out the pieces we want them to. Asking for the steps should be enough imo and it sounds like people agreed at our team meeting.

Reply:
Sounds good. We will have to clarify for Divya which systems to test on. But that's probably better than just listing whatever someone happened to test with anyway.